addons/twelve-hour-time/addon.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "12 hour time",
+  "description": "Changes forum times into the 12 hour time equivalent",
+  "tags": ["community", "forums"],
+  "enabledByDefault": false,
+  "credits": [
+    {
+      "name": "god286/webdev03",
+      "link": "https://github.com/webdev03"
+    },
+    {
+      "name": "JaydenDev",
+      "link": "https://github.com/JaydenDev"
+    },
```

Comment:
the credits section is for people who wrote code. not the person who just got the idea.